### DISCUSSION TITLE
adjusting frequency of requests

### DISCUSSION THREAD

**Discussion by protrading104** [Leo4j/Invoke-ADEnum#4] [Ideas]
how the script behaves in context of stealth from SOC ?

is it possible in the future to add the possibility of "adjusting frequency of requests"?

**Comment by Leo4j** [comment]
The tool is currently designed to speed-up the collection process, and not for stealth.

If the goal is to covertly enumerate a domain, then you better off running single commands and manually map the domain - so that your traffic would blend with other network traffic
